lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -218,6 +195,9 @@ private void init() {
     }
   }
 
+  public void startContainerManager() {
```

Comment:
Does this have to be `public`? I assume it is only called from within the local runtime code.